index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Survey Form</h1>
+    <p id="description">A simple survey form</p>
+<div id="form">
+    <form action="" id="survey-form">
+      <label for="name" id="name-label">Enter your name</label>
+      <input
+        required
+        type="text"
+        name="name"
+        id="name"
+        placeholder="Full name"
+      />
```

Comment:
You should always use semantic tags since it's the industry standard and also for accessibility features